includes/util.py

```
@@ -0,0 +1,89 @@
+    dst.close()
+
+def runcommand(command, env=None, stoponfail=1, silent=1, output=os.devnull):
+    ''' Run command, and handle special cases '''
+    env = env if env else None
+    args = shlex.split(command, posix=sys.platform != 'win32')
+    sp_args = {}
+    if sys.platform == 'win32':
+        sp_args['creationflags'] = subprocess.HIGH_PRIORITY_CLASS
+    if silent == 1:
+        devnull = open(output, 'w')
+        retval = subprocess.call(args,env=env,stdout=devnull,**sp_args)
```

Ruff (0.14.13)

`subprocess` call: check for execution of untrusted input

(S603)